Error in <TChain::LoadTree>: Cannot find tree with name analysistree/anatree in file /pnfs/uboone/data/uboone/root-tuple/anatree_outextbnb/anatree/prod_v05_08_00/00/00/53/65/ana_hist_28e166e8-364f-4b9f-86e1-147577c253b6.root
Error in <TChain::LoadTree>: Cannot find tree with name analysistree/anatree in file /pnfs/uboone/data/uboone/root-tuple/anatree_outextbnb/anatree/prod_v05_08_00/00/00/53/63/ana_hist_0466f0ac-c679-40ed-b166-178ee845e45e.root
Error in <TChain::LoadTree>: Cannot find tree with name analysistree/anatree in file /pnfs/uboone/data/uboone/root-tuple/anatree_outextbnb/anatree/prod_v05_08_00/00/00/53/38/ana_hist_1068a83b-7b61-4647-b21b-db6164c04a8a.root
Error in <TChain::LoadTree>: Cannot find tree with name analysistree/anatree in file /pnfs/uboone/data/uboone/root-tuple/anatree_outextbnb/anatree/prod_v05_08_00/00/00/52/04/ana_hist_20228886-aa1e-474b-8478-f2bcd4ded044.root
Error in <TChain::LoadTree>: Cannot find tree with name analysistree/anatree in file /pnfs/uboone/data/uboone/root-tuple/anatree_outextbnb/anatree/prod_v05_08_00/00/00/51/91/ana_hist_36bd29c4-970b-4751-bbe1-c0cf5cd9f887.root
Error in <TChain::LoadTree>: Cannot find tree with name analysistree/anatree in file /pnfs/uboone/data/uboone/root-tuple/anatree_outextbnb/anatree/prod_v05_08_00/00/00/59/56/ana_hist_c7807da6-e525-409d-8cd4-822b3daae07b.root
Error in <TChain::LoadTree>: Cannot find tree with name analysistree/anatree in file /pnfs/uboone/data/uboone/root-tuple/anatree_outextbnb/anatree/prod_v05_08_00/00/00/56/91/ana_hist_8f755d18-e50e-4dc9-9731-e772977a3f7b.root
Error in <TChain::LoadTree>: Cannot find tree with name analysistree/anatree in file /pnfs/uboone/data/uboone/root-tuple/anatree_outextbnb/anatree/prod_v05_08_00/00/00/53/64/ana_hist_074f60ec-71bc-46f2-8b17-1b6325e35b05.root
Error in <TChain::LoadTree>: Cannot find tree with name analysistree/anatree in file /pnfs/uboone/data/uboone/root-tuple/anatree_outextbnb/anatree/prod_v05_08_00/00/00/51/85/ana_hist_d8e68d19-ba05-4109-9415-3046b0b5a00a.root
Error in <RootX11IOErrorHandler>: fatal X11 error (connection to server lost?!)

**** Save data and exit application ****

TClass::TClass:0: RuntimeWarning: no dictionary for class box is available
TClass::TClass:0: RuntimeWarning: no dictionary for class MyLArTools is available
Error in <TStreamerInfo::Build:>: PandoraNuTrack: MyLArTools* has no streamer or dictionary, data member lar_tools will not be saved
TStreamerInfo::Build::0: RuntimeWarning: PandoraNuTrack: box has no streamer or dictionary, data member "roi" will not be saved
[?1034hNamespace(ev_frac=1.0, print_mod=1000, verbose=1)

reading list of files...
[95m    [NORMAL]  [0m[95m<LoadData> [0mLoading data for Geometry...
     file=/nashome/e/ecohen/larlite/core/LArUtil/dat/larutil_microboone.root 
[95m    [NORMAL]  [0m[95m<LoadData> [0mLoading data for DetectorProperties...
     file=/nashome/e/ecohen/larlite/core/LArUtil/dat/larutil_microboone.root 
[95m    [NORMAL]  [0m[95m<LoadData> [0mLoading data for LArProperties...
     file=/nashome/e/ecohen/larlite/core/LArUtil/dat/larutil_microboone.root 
entry: 1
run: 327690,subrun: 186609,event: -655238927
goodflashidx( 1 entries):	36,	
[31mtrack -99999
~~~~~~~~~~~~~~~~~~~~~ [0m
start_pos: (169.217,28.1737,574.714), |start_pos| = 599.77
end_pos: (58.9972,-107.542,772.048), |end_pos| = 781.731
length: 263.746
momentum: 1160.28
distlenratio: 0.999612
theta: 0.708471
phi: -2.2205
start_dqdx: 327.079
end_dqdx: 516.938
tot_dqdx: 310091
[34mroi[0]: (1215,1860) => (1935,3839)[0m
[34mroi[1]: (1309,1860) => (1372,3839)[0m
[34mroi[2]: (1915,1860) => (2573,3839)[0m
[31mtrack -99999
~~~~~~~~~~~~~~~~~~~~~ [0m
start_pos: (34.8277,101.864,528.487), |start_pos| = 539.34
end_pos: (23.9381,-101.495,638.789), |end_pos| = 647.244
length: 233.956
momentum: 1106.28
distlenratio: 0.989946
theta: 0.807909
phi: -1.58826
start_dqdx: 159.101
end_dqdx: 339.046
tot_dqdx: 19669.2
[34mroi[0]: (925,1231) => (1696,1427)[0m
[34mroi[1]: (1104,1231) => (1507,1427)[0m
[34mroi[2]: (1761,1231) => (2128,1427)[0m
[31mtrack -99999
~~~~~~~~~~~~~~~~~~~~~ [0m
start_pos: (84.2048,78.9702,776.947), |start_pos| = 785.477
end_pos: (78.9073,-104.45,873.844), |end_pos| = 883.594
length: 213.79
momentum: 1068.02
distlenratio: 0.970622
theta: 1.08561
phi: -1.55104
start_dqdx: 0
end_dqdx: 0
tot_dqdx: 0
[34mroi[0]: (1405,2218) => (2096,2313)[0m
[34mroi[1]: (1487,2218) => (1855,2313)[0m
[34mroi[2]: (2589,2218) => (2912,2313)[0m
[31mtrack -99999
~~~~~~~~~~~~~~~~~~~~~ [0m
start_pos: (182.853,-84.2837,185.302), |start_pos| = 273.634
end_pos: (200.963,-34.5355,211.924), |end_pos| = 294.093
length: 59.2915
momentum: 678.541
distlenratio: 0.999448
theta: 1.15479
phi: 1.17989
start_dqdx: 0
end_dqdx: 0
tot_dqdx: -7.19461e+36
[34mroi[0]: (791,4083) => (890,4408)[0m
[34mroi[1]: (398,4083) => (586,4408)[0m
[34mroi[2]: (617,4083) => (706,4408)[0m
[31mtrack -99999
~~~~~~~~~~~~~~~~~~~~~ [0m
start_pos: (169.849,-2.12478,858.439), |start_pos| = 875.084
end_pos: (169.359,-13.8899,934.062), |end_pos| = 949.393
length: 84.7049
momentum: 763.998
distlenratio: 0.903534
theta: 0.154651
phi: -1.60716
start_dqdx: 0
end_dqdx: 0
tot_dqdx: 0
[34mroi[0]: (1775,3841) => (1935,3850)[0m
[34mroi[1]: (1757,3841) => (1849,3850)[0m
[34mroi[2]: (2861,3841) => (3113,3850)[0m
[31mtrack -99999
~~~~~~~~~~~~~~~~~~~~~ [0m
start_pos: (52.7169,-74.5438,363.992), |start_pos| = 375.268
end_pos: (55.1061,-106.533,304.804), |end_pos| = 327.553
length: 69.7967
momentum: 715.956
distlenratio: 0.964542
theta: 2.73344
phi: -1.3844
start_dqdx: 0
end_dqdx: 0
tot_dqdx: 0
[34mroi[0]: (1154,1748) => (1160,1791)[0m
[34mroi[1]: (533,1748) => (724,1791)[0m
[34mroi[2]: (1015,1748) => (1212,1791)[0m
[31mtrack -99999
~~~~~~~~~~~~~~~~~~~~~ [0m
start_pos: (169.17,28.1362,574.857), |start_pos| = 599.892
end_pos: (176.508,37.6108,560.665), |end_pos| = 588.995
length: 18.5744
momentum: 472.686
distlenratio: 0.999986
theta: 2.44193
phi: 0.911628
start_dqdx: 0
end_dqdx: 0
tot_dqdx: 0
[34mroi[0]: (1164,3838) => (1215,3969)[0m
[34mroi[1]: (1372,3838) => (1376,3969)[0m
[34mroi[2]: (1868,3838) => (1915,3969)[0m
[31mtrack -99999
~~~~~~~~~~~~~~~~~~~~~ [0m
start_pos: (84.4825,77.9814,783.729), |start_pos| = 792.117
end_pos: (84.319,83.2994,795.203), |end_pos| = 803.988
length: 12.6717
momentum: 420.089
distlenratio: 0.998128
theta: 0.546673
phi: 1.5665
start_dqdx: 0
end_dqdx: 0
tot_dqdx: 0
[34mroi[0]: (1419,2315) => (1423,2318)[0m
[34mroi[1]: (1864,2315) => (1898,2318)[0m
[34mroi[2]: (2612,2315) => (2650,2318)[0m
[31mtrack -99999
~~~~~~~~~~~~~~~~~~~~~ [0m
start_pos: (86.8572,40.9351,478.481), |start_pos| = 488.02
end_pos: (88.6012,27.62,475.929), |end_pos| = 484.893
length: 13.6841
momentum: 430.002
distlenratio: 0.998906
theta: 1.72803
phi: -1.48396
start_dqdx: 0
end_dqdx: 0
tot_dqdx: 0
[34mroi[0]: (1017,2360) => (1052,2392)[0m
[34mroi[1]: (1205,2360) => (1248,2392)[0m
[34mroi[2]: (1586,2360) => (1594,2392)[0m
[31mtrack -99999
~~~~~~~~~~~~~~~~~~~~~ [0m
start_pos: (183.821,-85.4023,187.25), |start_pos| = 275.946
end_pos: (187.522,-85.3438,195.596), |end_pos| = 284.088
length: 9.16951
momentum: 383.414
distlenratio: 0.995714
theta: 0.492627
phi: 0.19519
start_dqdx: 0
end_dqdx: 0
tot_dqdx: 3.13891e-43
[34mroi[0]: (897,4101) => (910,4167)[0m
[34mroi[1]: (398,4101) => (412,4167)[0m
[34mroi[2]: (623,4101) => (651,4167)[0m
-------------------------------

 *** Break *** segmentation violation



===========================================================
There was a crash.
This is the entire stack trace of all threads:
===========================================================

Thread 2 (Thread 0x7fb3f9af1700 (LWP 7789)):
#0  0x000000312580da00 in sem_wait () from /lib64/libpthread.so.0
#1  0x00007fb404eb5688 in PyThread_acquire_lock (lock=0x1af6a70, waitflag=<value optimized out>) at Python/thread_pthread.h:324
#2  0x00007fb404e7bf44 in PyEval_RestoreThread (tstate=0x271d4e0) at Python/ceval.c:359
#3  0x00007fb3f9af449f in floatsleep (self=<value optimized out>, args=<value optimized out>) at /home/paterno/tmp/tmp-prod3/python/v2_7_10/Linux64bit+2.6-2.12/src/Python-2.7.10/Modules/timemodule.c:959
#4  time_sleep (self=<value optimized out>, args=<value optimized out>) at /home/paterno/tmp/tmp-prod3/python/v2_7_10/Linux64bit+2.6-2.12/src/Python-2.7.10/Modules/timemodule.c:206
#5  0x00007fb404e81c30 in call_function (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:4035
#6  PyEval_EvalFrameEx (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:2681
#7  0x00007fb404e8374e in PyEval_EvalCodeEx (co=0x7fb404bfe9b0, globals=<value optimized out>, locals=<value optimized out>, args=<value optimized out>, argcount=1, kws=0x7fb404d0d068, kwcount=0, defs=0x0, defcount=0, closure=0x0) at Python/ceval.c:3267
#8  0x00007fb404e018a8 in function_call (func=0x7fb3fabd1050, arg=0x7fb3fabd5d50, kw=0x7fb3f9d105c8) at Objects/funcobject.c:526
#9  0x00007fb404dd2293 in PyObject_Call (func=0x7fb3fabd1050, arg=<value optimized out>, kw=<value optimized out>) at Objects/abstract.c:2529
#10 0x00007fb404e80335 in ext_do_call (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:4348
#11 PyEval_EvalFrameEx (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:2720
#12 0x00007fb404e820db in fast_function (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:4121
#13 call_function (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:4056
#14 PyEval_EvalFrameEx (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:2681
#15 0x00007fb404e820db in fast_function (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:4121
#16 call_function (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:4056
#17 PyEval_EvalFrameEx (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:2681
#18 0x00007fb404e8374e in PyEval_EvalCodeEx (co=0x7fb3fabdd930, globals=<value optimized out>, locals=<value optimized out>, args=<value optimized out>, argcount=1, kws=0x0, kwcount=0, defs=0x0, defcount=0, closure=0x0) at Python/ceval.c:3267
#19 0x00007fb404e017a1 in function_call (func=0x7fb3f9d12500, arg=0x7fb3fabd5cd0, kw=0x0) at Objects/funcobject.c:526
#20 0x00007fb404dd2293 in PyObject_Call (func=0x7fb3f9d12500, arg=<value optimized out>, kw=<value optimized out>) at Objects/abstract.c:2529
#21 0x00007fb404de48cf in instancemethod_call (func=0x7fb3f9d12500, arg=0x7fb3fabd5cd0, kw=0x0) at Objects/classobject.c:2602
#22 0x00007fb404dd2293 in PyObject_Call (func=0x7fb3fb43f6e0, arg=<value optimized out>, kw=<value optimized out>) at Objects/abstract.c:2529
#23 0x00007fb404e7b2f3 in PyEval_CallObjectWithKeywords (func=0x7fb3fb43f6e0, arg=0x7fb404d0d050, kw=<value optimized out>) at Python/ceval.c:3904
#24 0x00007fb404ebb392 in t_bootstrap (boot_raw=<value optimized out>) at ./Modules/threadmodule.c:614
#25 0x0000003125807aa1 in start_thread () from /lib64/libpthread.so.0
#26 0x00000031250e893d in clone () from /lib64/libc.so.6

Thread 1 (Thread 0x7fb404d4e700 (LWP 7748)):
#0  0x00000031250ac69d in waitpid () from /lib64/libc.so.6
#1  0x000000312503e609 in do_system () from /lib64/libc.so.6
#2  0x000000312503e940 in system () from /lib64/libc.so.6
#3  0x00007fb3fdf97ba4 in TUnixSystem::StackTrace() () from /grid/fermiapp/products/larsoft/root/v5_34_32/Linux64bit+2.6-2.12-e9-nu-prof/lib/libCore.so
#4  0x00007fb3fdf99cbc in TUnixSystem::DispatchSignals(ESignals) () from /grid/fermiapp/products/larsoft/root/v5_34_32/Linux64bit+2.6-2.12-e9-nu-prof/lib/libCore.so
#5  <signal handler called>
#6  calcAnaTree::LoopPanNuTracks() () at calcAnaTree.cxx:167
#7  0x00007fb3f0d6e131 in calcAnaTree::get_bdt_tools(int) () at calcAnaTree.cxx:15
#8  0x00007fb3f0d7108c in G__AnaTreesGBDTprotons_GBDTprotonsPackageDict_787_0_13(G__value*, char const*, G__param*, int) () at AnaTreesGBDTprotons_GBDTprotonsPackageDict.cxx:1309
#9  0x00007fb3fd4268ab in Cint::G__CallFunc::Execute(void*) () from /grid/fermiapp/products/larsoft/root/v5_34_32/Linux64bit+2.6-2.12-e9-nu-prof/lib/libCint.so
#10 0x00007fb3fe67590b in PyROOT::TBoolExecutor::Execute(Cint::G__CallFunc*, void*, bool) () from /grid/fermiapp/products/larsoft/root/v5_34_32/Linux64bit+2.6-2.12-e9-nu-prof/lib/libPyROOT.so
#11 0x00007fb3fe67af96 in PyROOT::TMethodHolder<PyROOT::TScopeAdapter, PyROOT::TMemberAdapter>::CallSafe(void*, bool) () from /grid/fermiapp/products/larsoft/root/v5_34_32/Linux64bit+2.6-2.12-e9-nu-prof/lib/libPyROOT.so
#12 0x00007fb3fe67b109 in PyROOT::TMethodHolder<PyROOT::TScopeAdapter, PyROOT::TMemberAdapter>::Execute(void*, bool) () from /grid/fermiapp/products/larsoft/root/v5_34_32/Linux64bit+2.6-2.12-e9-nu-prof/lib/libPyROOT.so
#13 0x00007fb3fe67ab6d in PyROOT::TMethodHolder<PyROOT::TScopeAdapter, PyROOT::TMemberAdapter>::operator()(PyROOT::ObjectProxy*, _object*, _object*, long, bool) () from /grid/fermiapp/products/larsoft/root/v5_34_32/Linux64bit+2.6-2.12-e9-nu-prof/lib/libPyROOT.so
#14 0x00007fb3fe67f2ea in PyROOT::(anonymous namespace)::mp_call(PyROOT::MethodProxy*, _object*, _object*) () from /grid/fermiapp/products/larsoft/root/v5_34_32/Linux64bit+2.6-2.12-e9-nu-prof/lib/libPyROOT.so
#15 0x00007fb404dd2293 in PyObject_Call (func=0x7fb3f0a75610, arg=<value optimized out>, kw=<value optimized out>) at Objects/abstract.c:2529
#16 0x00007fb404e80e3a in do_call (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:4253
#17 call_function (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:4058
#18 PyEval_EvalFrameEx (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:2681
#19 0x00007fb404e8374e in PyEval_EvalCodeEx (co=0x7fb404c05d30, globals=<value optimized out>, locals=<value optimized out>, args=<value optimized out>, argcount=0, kws=0x0, kwcount=0, defs=0x0, defcount=0, closure=0x0) at Python/ceval.c:3267
#20 0x00007fb404e83862 in PyEval_EvalCode (co=<value optimized out>, globals=<value optimized out>, locals=<value optimized out>) at Python/ceval.c:669
#21 0x00007fb404ea36d0 in run_mod (fp=0x1825a10, filename=<value optimized out>, start=<value optimized out>, globals=0x7fb404ce2168, locals=0x7fb404ce2168, closeit=1, flags=0x7ffd72c73000) at Python/pythonrun.c:1371
#22 PyRun_FileExFlags (fp=0x1825a10, filename=<value optimized out>, start=<value optimized out>, globals=0x7fb404ce2168, locals=0x7fb404ce2168, closeit=1, flags=0x7ffd72c73000) at Python/pythonrun.c:1357
#23 0x00007fb404ea38af in PyRun_SimpleFileExFlags (fp=0x1825a10, filename=0x7ffd72c74e9c "mac/calc_list_of_files.py", closeit=1, flags=0x7ffd72c73000) at Python/pythonrun.c:949
#24 0x00007fb404eb91c4 in Py_Main (argc=<value optimized out>, argv=<value optimized out>) at Modules/main.c:640
#25 0x000000312501ed5d in __libc_start_main () from /lib64/libc.so.6
#26 0x0000000000400649 in _start ()
===========================================================


The lines below might hint at the cause of the crash.
If they do not help you then please submit a bug report at
http://root.cern.ch/bugs. Please post the ENTIRE stack trace
from above as an attachment in addition to anything else
that might help us fixing this issue.
===========================================================
#6  calcAnaTree::LoopPanNuTracks() () at calcAnaTree.cxx:167
#7  0x00007fb3f0d6e131 in calcAnaTree::get_bdt_tools(int) () at calcAnaTree.cxx:15
===========================================================



 *** Break *** segmentation violation



===========================================================
There was a crash.
This is the entire stack trace of all threads:
===========================================================

Thread 2 (Thread 0x7fb3f9af1700 (LWP 7789)):
#0  0x000000312580da00 in sem_wait () from /lib64/libpthread.so.0
#1  0x00007fb404eb5688 in PyThread_acquire_lock (lock=0x1af6a70, waitflag=<value optimized out>) at Python/thread_pthread.h:324
#2  0x00007fb404e7bf44 in PyEval_RestoreThread (tstate=0x271d4e0) at Python/ceval.c:359
#3  0x00007fb3f9af449f in floatsleep (self=<value optimized out>, args=<value optimized out>) at /home/paterno/tmp/tmp-prod3/python/v2_7_10/Linux64bit+2.6-2.12/src/Python-2.7.10/Modules/timemodule.c:959
#4  time_sleep (self=<value optimized out>, args=<value optimized out>) at /home/paterno/tmp/tmp-prod3/python/v2_7_10/Linux64bit+2.6-2.12/src/Python-2.7.10/Modules/timemodule.c:206
#5  0x00007fb404e81c30 in call_function (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:4035
#6  PyEval_EvalFrameEx (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:2681
#7  0x00007fb404e8374e in PyEval_EvalCodeEx (co=0x7fb404bfe9b0, globals=<value optimized out>, locals=<value optimized out>, args=<value optimized out>, argcount=1, kws=0x7fb404d0d068, kwcount=0, defs=0x0, defcount=0, closure=0x0) at Python/ceval.c:3267
#8  0x00007fb404e018a8 in function_call (func=0x7fb3fabd1050, arg=0x7fb3fabd5d50, kw=0x7fb3f9d105c8) at Objects/funcobject.c:526
#9  0x00007fb404dd2293 in PyObject_Call (func=0x7fb3fabd1050, arg=<value optimized out>, kw=<value optimized out>) at Objects/abstract.c:2529
#10 0x00007fb404e80335 in ext_do_call (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:4348
#11 PyEval_EvalFrameEx (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:2720
#12 0x00007fb404e820db in fast_function (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:4121
#13 call_function (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:4056
#14 PyEval_EvalFrameEx (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:2681
#15 0x00007fb404e820db in fast_function (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:4121
#16 call_function (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:4056
#17 PyEval_EvalFrameEx (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:2681
#18 0x00007fb404e8374e in PyEval_EvalCodeEx (co=0x7fb3fabdd930, globals=<value optimized out>, locals=<value optimized out>, args=<value optimized out>, argcount=1, kws=0x0, kwcount=0, defs=0x0, defcount=0, closure=0x0) at Python/ceval.c:3267
#19 0x00007fb404e017a1 in function_call (func=0x7fb3f9d12500, arg=0x7fb3fabd5cd0, kw=0x0) at Objects/funcobject.c:526
#20 0x00007fb404dd2293 in PyObject_Call (func=0x7fb3f9d12500, arg=<value optimized out>, kw=<value optimized out>) at Objects/abstract.c:2529
#21 0x00007fb404de48cf in instancemethod_call (func=0x7fb3f9d12500, arg=0x7fb3fabd5cd0, kw=0x0) at Objects/classobject.c:2602
#22 0x00007fb404dd2293 in PyObject_Call (func=0x7fb3fb43f6e0, arg=<value optimized out>, kw=<value optimized out>) at Objects/abstract.c:2529
#23 0x00007fb404e7b2f3 in PyEval_CallObjectWithKeywords (func=0x7fb3fb43f6e0, arg=0x7fb404d0d050, kw=<value optimized out>) at Python/ceval.c:3904
#24 0x00007fb404ebb392 in t_bootstrap (boot_raw=<value optimized out>) at ./Modules/threadmodule.c:614
#25 0x0000003125807aa1 in start_thread () from /lib64/libpthread.so.0
#26 0x00000031250e893d in clone () from /lib64/libc.so.6

Thread 1 (Thread 0x7fb404d4e700 (LWP 7748)):
#0  0x00000031250ac69d in waitpid () from /lib64/libc.so.6
#1  0x000000312503e609 in do_system () from /lib64/libc.so.6
#2  0x000000312503e940 in system () from /lib64/libc.so.6
#3  0x00007fb3fdf97ba4 in TUnixSystem::StackTrace() () from /grid/fermiapp/products/larsoft/root/v5_34_32/Linux64bit+2.6-2.12-e9-nu-prof/lib/libCore.so
#4  0x00007fb3fdf99cbc in TUnixSystem::DispatchSignals(ESignals) () from /grid/fermiapp/products/larsoft/root/v5_34_32/Linux64bit+2.6-2.12-e9-nu-prof/lib/libCore.so
#5  <signal handler called>
#6  0x00007fb404e073eb in list_dealloc (op=0x7fb3f13db710) at Objects/listobject.c:309
#7  0x00007fb404e0510e in listiter_dealloc (it=0x7fb3f0a74c50) at Objects/listobject.c:2888
#8  0x00007fb404e7c86f in PyEval_EvalFrameEx (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:2909
#9  0x00007fb404e8374e in PyEval_EvalCodeEx (co=0x7fb404c05d30, globals=<value optimized out>, locals=<value optimized out>, args=<value optimized out>, argcount=0, kws=0x0, kwcount=0, defs=0x0, defcount=0, closure=0x0) at Python/ceval.c:3267
#10 0x00007fb404e83862 in PyEval_EvalCode (co=<value optimized out>, globals=<value optimized out>, locals=<value optimized out>) at Python/ceval.c:669
#11 0x00007fb404ea36d0 in run_mod (fp=0x1825a10, filename=<value optimized out>, start=<value optimized out>, globals=0x7fb404ce2168, locals=0x7fb404ce2168, closeit=1, flags=0x7ffd72c73000) at Python/pythonrun.c:1371
#12 PyRun_FileExFlags (fp=0x1825a10, filename=<value optimized out>, start=<value optimized out>, globals=0x7fb404ce2168, locals=0x7fb404ce2168, closeit=1, flags=0x7ffd72c73000) at Python/pythonrun.c:1357
#13 0x00007fb404ea38af in PyRun_SimpleFileExFlags (fp=0x1825a10, filename=0x7ffd72c74e9c "mac/calc_list_of_files.py", closeit=1, flags=0x7ffd72c73000) at Python/pythonrun.c:949
#14 0x00007fb404eb91c4 in Py_Main (argc=<value optimized out>, argv=<value optimized out>) at Modules/main.c:640
#15 0x000000312501ed5d in __libc_start_main () from /lib64/libc.so.6
#16 0x0000000000400649 in _start ()
===========================================================


The lines below might hint at the cause of the crash.
If they do not help you then please submit a bug report at
http://root.cern.ch/bugs. Please post the ENTIRE stack trace
from above as an attachment in addition to anything else
that might help us fixing this issue.
===========================================================
#6  0x00007fb404e073eb in list_dealloc (op=0x7fb3f13db710) at Objects/listobject.c:309
#7  0x00007fb404e0510e in listiter_dealloc (it=0x7fb3f0a74c50) at Objects/listobject.c:2888
#8  0x00007fb404e7c86f in PyEval_EvalFrameEx (f=<value optimized out>, throwflag=<value optimized out>) at Python/ceval.c:2909
#9  0x00007fb404e8374e in PyEval_EvalCodeEx (co=0x7fb404c05d30, globals=<value optimized out>, locals=<value optimized out>, args=<value optimized out>, argcount=0, kws=0x0, kwcount=0, defs=0x0, defcount=0, closure=0x0) at Python/ceval.c:3267
#10 0x00007fb404e83862 in PyEval_EvalCode (co=<value optimized out>, globals=<value optimized out>, locals=<value optimized out>) at Python/ceval.c:669
#11 0x00007fb404ea36d0 in run_mod (fp=0x1825a10, filename=<value optimized out>, start=<value optimized out>, globals=0x7fb404ce2168, locals=0x7fb404ce2168, closeit=1, flags=0x7ffd72c73000) at Python/pythonrun.c:1371
#12 PyRun_FileExFlags (fp=0x1825a10, filename=<value optimized out>, start=<value optimized out>, globals=0x7fb404ce2168, locals=0x7fb404ce2168, closeit=1, flags=0x7ffd72c73000) at Python/pythonrun.c:1357
#13 0x00007fb404ea38af in PyRun_SimpleFileExFlags (fp=0x1825a10, filename=0x7ffd72c74e9c "mac/calc_list_of_files.py", closeit=1, flags=0x7ffd72c73000) at Python/pythonrun.c:949
#14 0x00007fb404eb91c4 in Py_Main (argc=<value optimized out>, argv=<value optimized out>) at Modules/main.c:640
#15 0x000000312501ed5d in __libc_start_main () from /lib64/libc.so.6
#16 0x0000000000400649 in _start ()
===========================================================


